gammapy/scripts/jupyter.py

```diff
@@ -0,0 +1,144 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Command line tool to perform devel management actions on jupyter notebooks."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+from black import format_str
+import click
+import logging
+import nbformat
+import subprocess
+import sys
+import testipynb
+import time
+from unittest import TestCase
+from ..extern.pathlib import Path
+
+
+log = logging.getLogger(__name__)
+
+
+@click.command(name='black')
+@click.pass_context
+def cli_jupyter_black(ctx):
+    """Format code cells with black."""
+
+    for notebook in build_nblist(ctx):
```

Comment:
What you call `notebook` here are Path objects, not Notebook objects?
Maybe rename the variable to "path"?